gallery/build.gradle

```diff
@@ -85,3 +48,4 @@ dependencies {
     // Image loader
     implementation 'com.github.bumptech.glide:glide:4.7.1'
 }
+apply from: 'publishing.gradle'
```

Comment:
Why in the end of file?

Reply:
This `apply` should be after `android` block to get compileSdkVersion, so i decided to move it in the end of file.